src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -236,6 +242,38 @@ export default {
 			// eslint-disable-next-line no-underscore-dangle
 			return `${this.loans[this.detailedLoanIndex].__typename}:${this.loans[this.detailedLoanIndex].id}`;
 		},
+		noHoverLoan() {
+			return this.hoverLoanIndex === null;
+		},
+		hoverLoanIsLeftMost() {
+			if (this.noHoverLoan) {
+				return false;
+			}
+
+			const bodyRect = document.body.getBoundingClientRect();
+			const hoverLoanCardRect = this.$refs.hoverLoanCards[this.hoverLoanIndex].$el.getBoundingClientRect();
+			const hoverLoanCardLeft = hoverLoanCardRect.left - bodyRect.left;
+
+			const hoverCardsHolderRect = this.$refs.hoverCardsHolder.getBoundingClientRect();
+			const noArrowPaddingSide = 1;
+			const arrowWidthInRem = 2.5;
+			const pxInRem = 16;
+			const leftPadding = this.categoryRowArrowsVisible()
+				? arrowWidthInRem * pxInRem
+				: noArrowPaddingSide * pxInRem;
+			const hoverCardsHolderLeft = hoverCardsHolderRect.left - bodyRect.left + leftPadding;
+
+			const hoverCardDistanceFromLeft = hoverLoanCardLeft - hoverCardsHolderLeft;
+
+			return hoverCardDistanceFromLeft === 0;
```

Comment:
Is 0 the only distance that qualifies or is there a range of distances that are ok? For instance if the distance is less than 1 rem.

Reply:
Interesting question. I believe the scrolling behavior we use (scrolling by a multiple of the width of a card + margin-right) makes it so that it should always be zero but let me think about this and test it more. We can easily make this more lax if there’s a chance of outliers.

Reply:
Rereading the code there shouldn't be a non-zero case. I'll continue to keep an eye on this and test :+1: